etna/experimental/classification/feature_extraction/weasel.py

```diff
@@ -0,0 +1,374 @@
+from copy import deepcopy
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+import numpy as np
+from numpy.lib.stride_tricks import sliding_window_view
+from pyts.approximation import SymbolicFourierApproximation
+from pyts.transformation import WEASEL
+from scipy.sparse import coo_matrix
+from scipy.sparse import csr_matrix
+from scipy.sparse import hstack
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_selection import chi2
+from typing_extensions import Literal
+
+from etna.experimental.classification.feature_extraction.base import BaseTimeSeriesFeatureExtractor
+from etna.experimental.classification.utils import padd_single_series
+
+
+class CustomWEASEL(WEASEL):
+    """Improved version of WEASEL transform to work with the series of different length."""
+
+    def __init__(
```

Comment:
Can we specify something with default parameters?

Reply:
We specify all the defaults in `WEASELFeatureExtractor`